postgres-appliance/scripts/configure_spilo.py

```diff
@@ -596,7 +596,7 @@ def write_wale_environment(placeholders, prefix, overwrite):
         if not (wale.get('AWS_SECRET_ACCESS_KEY') and wale.get('AWS_ACCESS_KEY_ID')):
             wale['AWS_INSTANCE_PROFILE'] = 'true'
             write_envdir_names.append('AWS_INSTANCE_PROFILE')
-        if wale.get('USE_WALG_BACKUP') and not wale.get('WALG_S3_SSE'):
+        if wale.get('USE_WALG_BACKUP') and not wale.get('WALG_S3_SSE') and wale.get('WALG_DISABLE_S3_SSE') is None:
```

Comment:
Its not gonna work like this. When the value is not defined line 556 will set it to empty string

Reply:
Ok, I'll change it.

Reply:
@CyberDem0n Please take a look at the updated file.